src/CMakeLists.txt

```diff
@@ -17,13 +17,12 @@ else()
   message(FATAL_ERROR "Failed to determine the source files for the regular expression backend")
 endif()
 
+add_library(benchmark ${SOURCE_FILES} ${RE_FILES})
+
 # Build the benchmark library
```

Comment:
This comment would make more sense on line 19. A more apt comment would be `# Link threading if building shared library`